jvcl/run/JvStrToHtml.pas

```diff
@@ -10,8 +10,8 @@
 
 The Original Code is: JvStrToHtml.PAS, released on 2001-02-28.
 
-The Initial Developer of the Original Code is S�bastien Buysse [sbuysse att buypin dott com]
```

Comment:
This should not have been changed by your commit, leading me to think that the file encoding has been changed